### PR TITLE
go to oe-alliance new version

### PR DESCRIPTION
**Pull request by fairbird**

### CODE REVIEW

**Comment by eriksl** [comment]
At least make the SUBJECT informational. It doesn't mention "minisatip" or "satipclient" anywhere.

File: meta-openpli/recipes-openpli/satip/satipclient.bb [hunk 1]
```diff
@@ -28,4 +33,5 @@ do_install_append() {
 
 EXTRA_OECONF = " \
     --with-boxtype=${MACHINE} \
+    --with-machinebuild="${MACHINEBUILD}" \
```
Comment (athoik) [inline meta-openpli/recipes-openpli/satip/satipclient.bb:36]: Where in OpenPLi OE MACHINEBUILD is defined?

File: meta-openpli/recipes-openpli/satip/minisatip.bb [hunk 2]
```diff
@@ -1,20 +1,35 @@
...
-FILES_${PN} = "/usr/sbin/minisatip"
 S = "${WORKDIR}/git"
-BUILD = "${WORKDIR}/build"
+BUILD = "${WORKDIR}/git"
+
+inherit autotools-brokensep
 
-inherit autotools
+INITSCRIPT_NAME = "minisatip"
+EXTRA_OECONF = "--enable-enigma --disable-netcv"
```
Comment (athoik) [inline meta-openpli/recipes-openpli/satip/minisatip.bb:23]: Enable Enigma requires SAT>IP branch on Enigma2, right?

**Comment by fairbird** [comment]
sorry my friend I have forget to write correct (SUBJECT)
It should update minisatip ...

**Comment by persianpros** [comment]
This merge request won't work.

**Comment by fairbird** [comment]
I have built it after remove this (as athoik wrote) to DM800HD and DM800se
` --with-machinebuild="${MACHINEBUILD}" \`
and I have upload it on my images feed .
And here is some confirm from some friend he asked me before to up date minisatip to 3.1d

> Yes, I confirm it.  Yesterday downloaded and installed without any problem.  Work perfecty the tuner served to tvheadend.
Thank you very much.

**Comment by persianpros** [comment]
Yes I mean you need to edit it and remove that line otherwise this merge request won't work.

**Comment by fairbird** [comment]
Yes exactly my friend  .

________________________________

**Comment by WanWizard** [comment]
Closed due to lack of futher response.

**Pull request closed without merging**